pkg/controller/constraint/constraint_controller.go

```diff
@@ -111,6 +141,23 @@ func (r *ReconcileConstraint) Reconcile(request reconcile.Request) (reconcile.Re
 		return reconcile.Result{}, err
 	}
 
+	constraintKey := strings.Join([]string{instance.GetKind(), instance.GetName()}, "/")
+	enforcementAction, err := util.GetEnforcementAction(instance.Object)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	r.constraintsCache.addConstraintKey(constraintKey, tags{
```

Comment:
Why are we updating the constraints cache here? No processing has been done yet.

Reply:
moved after update for active scenario